README.md

```diff
@@ -41,22 +41,22 @@ const cancelStuckTransactions = new CancelStuckTransactions({
 // Start the cancel transactions loop
 ;(async () => {
   while (true) {
-    await cancelStuckTransactions.olderThan(TEN_MINUTES)
+    await stuckTransactionsCanceller.cancelOlderThan(TEN_MINUTES)
     await timers.setTimeout(ONE_MINUTE)
   }
 })()
 
 // Create a transaction somehow
 const tx = await ethers.createTransaction(/* ... */)
 
-// After you create a transactions, set it as pending
-cancelStuckTransactions.pending(tx)
+// After you create a transactions, add it as pending
+stuckTransactionsCanceller.addPending(tx)
 
 // Start waiting for confirmations
 await tx.wait()
 
-// Once confirmed, set it as successful
-await cancelStuckTransactions.successful(tx)
+// Once confirmed, remove it
+await stuckTransactionsCanceller.removeSuccessful(tx)
```

Comment:
The transaction does not necessarilly must be successful, right? If the smart contract rejects the method invocation in the TX, the transaction is still confirmed and added to the blockchain, and therefore we should remove it from the list of pending transactions.

Proposed name: `removeConfirmed` or `removeCommitted`.